src/app/lib/notion/projects.ts

```diff
@@ -0,0 +1,66 @@
+const cache = new Map();
+const cacheTTL = 0 //24 * 60 * 60 * 1000;
+
+async function getProjects() {
+  const cacheKey = 'notion_projects';
+  const now = Date.now();
+  const notionDatabaseId = process.env.NOTION_DB_ID
+  const notionApiKey = process.env.NOTION_DB_SECRET
+  
+  
+  if (cache.has(cacheKey)) {
+    const { data, timestamp } = cache.get(cacheKey);
+    if (now - timestamp < cacheTTL) {
+      return data;
+    }
+  }
+
+  const filter = {
+    "filter": {
+      "or": [
+      {
+        "property": "Year",
+        "multi_select": {
+          "contains": "2023"
+        }
+      },
+      {
+        "property": "Year",
+        "multi_select": {
+          "contains": "2022"
+        }
+      }
+    ]
+    }
+  }
+
```

Comment:
let's add a tag on that page as "starred" and just get those and not based on year?